lib/eunit/test/tutf8.erl

```diff
@@ -0,0 +1,12 @@
+%% coding: utf-8
+
+-module(tutf8).
+
+-include_lib("eunit/include/eunit.hrl").
+
+foo_test_() ->
+	[
+ 	  {<<"_göod">>, fun() -> io:format("goöd"), io:format([128,64]), ?assert("gö"=="gö") end}
+ 	 ,{"_bäd", fun() -> io:format("båad"), io:format([128,65]), ?assert("wå"=="wä") end}
+	 ,{"x汉x", fun() -> io:format("_字_"), io:format([128,66]), ?assert("汉"=="汉") end}
```

Comment:
But the first test/line is "-göod" is included in the generated xml file even if it succeeds?